sdk/textanalytics/Azure.AI.TextAnalytics/src/DocumentHealthcareEntitiesCollection.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Collection of <see cref="DocumentHealthcareEntities"/> objects in a document.
+    /// </summary>
+    public class DocumentHealthcareEntitiesCollection : ReadOnlyCollection<DocumentHealthcareEntities>
```

Comment:
you could either remove this class for this preview and have the `StartAnalyzeHealthcare` endpoint return a list of `DocumentHealthcareEntities` or look for a way to avoid duplication